src/librustdoc/html/format.rs

```diff
@@ -947,8 +950,14 @@ impl clean::FnRetTy {
 }
 
 impl clean::BareFunctionDecl {
-    fn print_generic_params(&self) -> impl fmt::Display + '_ {
-        comma_sep(self.generic_params.iter().map(|g| g.print()))
+    fn print_hrtb_bounds_with_space(&self) -> impl fmt::Display + '_ {
```

Comment:
nit: 'higher ranked trait bounds bounds' is redundant.
```suggestion
    fn print_hrtb_with_space(&self) -> impl fmt::Display + '_ {
```